README.md

```diff
@@ -368,5 +382,44 @@ const toast2 = showToast("number 2", configs);
 ```
 
 ### Styling the toast
-
 TODO
+
+## Security and Privacy Considerations
+See [TAG Security / Privacy Self Review](/security-privacy-self-review.md).
+
+## Considered Alternatives
+
+### Extending the Notification API
+Toasts are intended to be ephemeral, context-specific messages,
+and collecting them in a user's notifications tray doesn't reflect the spirit of the pattern.
+[kenchris]() put this well in [this comment]() on the TAG Design Review:
+
+> the difference is that this is a temporary notification / infobar, 
+> like you see for PWAs ("new version available, press reload to refresh") 
+> or like in GMail ("Chat is currently unavailable") etc. 
+> These are not things you want an actual system notification for... 
+> They really depend on the surrounding content/situation unlike regular notifications.
+
+Toasts also often contain some sort of action the user can take which affects the state of the page 
+(e.g. un-sending an email on Gmail).
+To provide this kind of behavior accessibly the element should live wholly in the page / DOM.
```

Comment:
I'm not sure these two sentences are accurate. Notifications can have actions these days, and I assume the OS vendors have made their notifications accessible.

Reply:
Yeah I had the wrong impression initially, updated to reflect.